pandas/core/generic.py

```diff
@@ -10984,8 +10992,16 @@ def stat_func(
 
 
 def _make_stat_function(
-    cls, name, name1, name2, axis_descr, desc, f, see_also: str = "", examples: str = ""
-):
+    cls,
+    name: str,
+    name1: str,
+    name2: str,
+    axis_descr: str,
+    desc: str,
+    func: Callable,
```

Comment:
Is it possible to add subtypes here or no?

Reply:
The passed functions don't have the same signature, because e.g. ``nansum`` has a min_count`` parameter and ``nankurt`` does not. So it's not possible without splitting the function into two, which IMO is not worth it.